easybuild/easyconfigs/p/PySide2/PySide2-5.14.2.3-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'PythonPackage'
+
+name = 'PySide2'
+version = '5.14.2.3'
+
+homepage = "https://pypi.python.org/pypi/%(name)s/"
+description = """PySide2 is the official Python module from the Qt for Python project,
+    which provides access to the complete Qt 5.12+ framework."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = ['https://download.qt.io/official_releases/QtForPython/%(namelower)s/%(name)s-%(version)s-src/']
+sources = ['pyside-setup-opensource-src-%(version)s.tar.xz']
+checksums = ['032de309505c78ba52c225aaf5fd717b8d87ae863aa7cee8ba58ba6141c59022']
+
+builddependencies = [
+    ('CMake', '3.18.4'),
+    ('Sphinx', '3.5.2'),
+    ('Clang', '11.0.1'),
+]
```

Comment:
@SimonPinches For `GCCcore`, you need to include `binutils` as a builddep. In this case, version `2.34`.

Reply:
@verdurin I think it should be 2.35.

Reply:
Done :-)